sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -30,7 +30,7 @@ Azure::Response<CertificateOperationProperties> CreateCertificateOperation::Poll
 std::unique_ptr<Azure::Core::Http::RawResponse> CreateCertificateOperation::PollInternal(
     Azure::Core::Context const& context)
 {
-  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse;
+  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse = std::move(m_rawResponse);
```

Comment:
I'm a bit confused by this. By my reading, we move the member `m_rawResponse` to the `rawResponse` local variable at line 33 and then immediately overwrite it at line 37.

If the intent is to free `m_rawResponse`, then why not juse call `m_rawResponse.release()`, which would seem to be a more straightforward way of implementing that.

Reply:
Yeah, so was I.

It looks like such a change is only needed for deleted_key_operation and recover_deleted_key_operation, which have an `if (!IsDone())` block. The other LRO impls don't.

https://github.com/Azure/azure-sdk-for-cpp/pull/3952#discussion_r975684933